### PR TITLE
Block out Menus

### PR DESCRIPTION
**Pull request by tabascq**
Laying down some basic menu structure for inspiration. Open to change,
just helping to define the rough shape of things.

Linked to Issue #53.

### CODE REVIEW

File: ServerCore/Pages/_Layout.cshtml [hunk 1]
```diff
@@ -15,8 +15,16 @@
...
     </environment>
+
+    <!-- TODO: Remove when menus are implemented -->
+    <style>
+        .nav i { color: gray; font-size: 10px; }
+    </style>
 </head>
 <body>
+    @{ 
+        var Event = ViewData["Event"] as ServerCore.DataModel.Event;
```
Comment (morganbr) [inline ServerCore/Pages/_Layout.cshtml]: Is it necessary to use ViewData in layouts or is there some way to reference the model?

File: ServerCore/Pages/_Layout.cshtml [hunk 2]
```diff
@@ -26,15 +34,91 @@
...
                 <ul class="nav navbar-nav">
                     <li><a asp-page="/Index">Home</a></li>
-                    <li><a asp-page="/Puzzleday/Index">Puzzleday</a></li>
-                    <li><a asp-page="/Puzzlehunt/Index">Puzzlehunt</a></li>
-                    <li><a asp-page="/About">About</a></li>
-                    <li><a asp-page="/Contact">Contact</a></li>
+
+                    @if (Event != null)
+                    {
+                    <li class="dropdown">
```
Comment (morganbr) [inline ServerCore/Pages/_Layout.cshtml]: Should we indent from the curly brace @asyasky?

File: ServerCore/Pages/_Layout.cshtml (hunk 1 above)
Comment (tabascq) [reply ServerCore/Pages/_Layout.cshtml]: The book I have says ViewData is needed for things in the shared file. With more engineering I am sure there is something more strongly typed but I am not sure it is worth it. I do have plans to automatically fill in the Event via routing and other things, which will help somewhat.

**Comment by tabascq** [comment]
Screenshots are a bit of a chore with dropdowns but in general I get your point :-)

**Comment by tabascq** [comment]
And thanks for the review!

File: ServerCore/Pages/_Layout.cshtml (hunk 2 above)
Comment (asyasky) [reply ServerCore/Pages/_Layout.cshtml]: Yes please. :) An automatic style sheet is on my todo list, but in the mean time ctrl+k+d fixes up most things (not sure about cshtml, but it does in cs for sure).

**Pull request merged**